deployment.rb

```diff
@@ -69,6 +69,8 @@ def load_configuration()
     'read_only'                   => false,
     'ruby_installer'              => rack_env == :development ? 'rbenv' : 'system',
     'root_dir'                    => root_dir,
+    'use_dynamo_tables'           => [:staging, :production].include?(rack_env),
```

Comment:
I'm going to disable this for production for now, while we figure out exactly what our range keys are going to be, to avoid a potential migration for applab data. Applab does have some users whose data we need to preserve, namely the writers for the CSP course.